docs/source/create_channel/create_channel.md

```diff
@@ -364,7 +376,7 @@ Blockchain info: {"height":3,"currentBlockHash":"eBpwWKTNUgnXGpaY2ojF4xeP3bWdjlP
 
 We can confirm that the channel was created successfully by deploying a chaincode to the channel. We can use the `network.sh` script to deploy the Basic asset transfer chaincode to any test network channel. Deploy a chaincode to our new channel using the following command:
 ```
-./network.sh deployCC --ccn basic -ccp ../asset-transfer-basic/chaincode-go/ -ccl go -c channel1 --cci InitLedger 
+./network.sh deployCC --ccn basic -ccp ../asset-transfer-basic/chaincode-go/ -ccl go -c channel1 --cci InitLedger
```

Comment:
Just hope this shell will not invoke any 2.3 new added binary.
https://github.com/hyperledger/fabric-samples/blob/14dc7e13160ef1b7332bafb01f8ffa865116f9e7/test-network/scripts/createChannel.sh#L40
or not invoke any new api added in 2.3 as `:7053/participation/v1/channels`

Reply:
It will not. We cut the 2.2.1 tag of samples prior to merging the channel participation changes. This PR is to specifically download the 2.2.1 test network artifacts and not the 2.3.0 artifacts that have the participation API in them.

Reply:
Aha! cool.